src/components/common/MaintenanceBar.vue

```diff
@@ -1,23 +1,36 @@
 <template>
-  <div v-if="session.maintenanceBar && show" class="maintenance-bar">
-    <i></i>
-    <p>
-      We've identified problems with our servers that are causing issues for
-      some of our users. We apologize for the disruption and are working on a
-      fix.
-    </p>
-    <a class="close">
-      <i class="material-icons" @click="close">close</i>
-    </a>
+  <div>
+    <div v-if="session.maintenanceBar && show" class="maintenance-bar">
```

Comment:
Do we still need this switch? This should now be obsolete as we show the maintenance bar only of there are records in the DB?

Reply:
This is also replicating the same html structure twice. I propose we remove this section completely. Remember to also remove the `session.maintenanceBar` in the store